### PR TITLE
refactor: remove compiler warnings

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-1924

JIRA issue URL: *paste URL here*

*[Add a short description of what is changed here.]*

## Reviewer tasks

The following is based on the
[Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines).
As a reviewer, you should check the guidelines regularly for updates, and just
to refresh your memory.

The reviewer can use this list for reference to make sure
they have considered all the important points.

Use the checkboxes or not, whatever works best for you.

 - Code quality
   - [ ] Code passes style check (checkstyle/eslint)
   - [ ] Code is clear and concise
   - [ ] UI code is internationalised
   - [ ] Code has adequate comments where appropriate
   - Code is in an appropriate place
     - [ ] Classes are in appropriate packages
     - [ ] Code fits with class's responsibilities (SRP)
   - [ ] Configuration files are documented enough
   - If code is removed
     - [ ] No remaining code references the removed code
       - [ ] No orphaned rewrite rules
       - [ ] No orphaned config settings
     - [ ] No remaining comments refer to the removed code
     - [ ] Unused imports and libraries are removed
 - Testing
   - [ ] New code has tests
   - [ ] Changed code has tests
   - [ ] All tests pass
   - [ ] Test coverage stable or increased
 - Documentation
   - [ ] New features are documented
   - [ ] Docs removed for deleted features
   - [ ] Docs updated for all changed features
   - [ ] Developer/sysadmin docs updated if appropriate
 - If there are new dependencies
   - [ ] Does each new dependency pass all the
         [Considerations for new dependencies](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines#new-technologies-and-dependencies)?

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/324)

### CODE REVIEW

**Comment by huangp** [comment]
:lgtm: -(Awaiting review) +Reviewed +(On QA) 

---

Reviewed 87 of 88 files at r1, 14 of 14 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-platform/324#-:-KkTUvymy-o3T7pPYpmA:b49ms4)*

**Comment by djansen-redhat** [comment]
101 files with potentially unrelated changes? Dude. Seriously.

**Comment by djansen-redhat** [comment]
Test List
- [ ] Hours remaining still works
- [ ] Client account query still works
- [ ] Client glossary query still works
- [ ] Client project/iteration query still works
- [ ] Translated docs show correctly in gwt editor

File: server/zanata-model/src/main/java/org/zanata/hibernate/search/AbstractFieldBridge.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import org.apache.lucene.document.Field;
+import org.apache.lucene.document.FieldType;
+import org.apache.lucene.index.IndexOptions;
+import org.hibernate.search.bridge.FieldBridge;
+import org.hibernate.search.bridge.LuceneOptions;
+
+/**
+ * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
+ */
+public abstract class AbstractFieldBridge implements FieldBridge {
```
Comment (seanf) [inline server/zanata-model/src/main/java/org/zanata/hibernate/search/AbstractFieldBridge.java]: This Lucene work really deserves a separate pull request.

Comment (aeng) [inline server/zanata-model/src/main/java/org/zanata/hibernate/search/AbstractFieldBridge.java]: Don't think I will make anymore changes in this PR. But I will make sure the subsequent PR will be per warning.

**Comment by djansen-redhat** [comment]
Seems ok
